src/api/services/CryptocurrencyAddressService.ts

```diff
@@ -37,9 +37,13 @@ export class CryptocurrencyAddressService {
 
     @validate()
     public async create( @request(CryptocurrencyAddressCreateRequest) body: CryptocurrencyAddressCreateRequest): Promise<CryptocurrencyAddress> {
-
         // If the request body was valid we will create the cryptocurrencyAddress
-        const cryptocurrencyAddress = await this.cryptocurrencyAddressRepo.create(body);
+        let cryptocurrencyAddress;
+        await this.cryptocurrencyAddressRepo.create(body).catch(e => {
+            this.log.error('CryptocurrencyAddressService.create():100: ' + e);
+        }).then(res => {
+            cryptocurrencyAddress = res;
+        });
         // this.log.debug('CryptocurrencyAddressService created: ', JSON.stringify(cryptocurrencyAddress.toJSON(), null, 2));
```

Comment:
const cryptocurrencyAddress = await this.cryptocurrencyAddressRepo.create(body).catch(e => {
    this.log.error('CryptocurrencyAddressService.create(): ', e);
    throw some error if we cant continue
});